Tests/Tests/AppStartEventSpec.swift

```diff
@@ -6,11 +6,12 @@ import Nimble
 class AppStartEventSpec: QuickSpec {
     override func spec() {
         describe("AppStartEvent") {
-            let appStartEvent = AppStartEvent.init(timestamp: 50)
+            let appStartEvent = AppStartEvent(timestamp: 50)
             context("AppStartEvent.analyticsParameters") {
                 it("will return dictionary with eventName and timestamp values") {
                     expect(appStartEvent.analyticsParameters).toNot(beNil())
-                    expect(appStartEvent.analyticsParameters).to(beAKindOf([String: Any].self))
+                    expect(appStartEvent.analyticsParameters["eventName"]).to(beAKindOf(String.self))
```

Comment:
I meant to expect eventName to be `equal("appStart")` and for timestamp value to be `50`.
Please make similar changes in other test cases. There's no need to check the type